### DISCUSSION TITLE
Adding information about individual publications

### DISCUSSION THREAD

**Discussion by nishadsinghi** [alshedivat/al-folio#1951] [Q&A] [answered]
For the publications page, I would love to be able to add some additional information about individual papers (for example, a one-line summary; press; oral award; other awards). Some of these already seem to be present on @alshedivat's webpage. How can I add this information to my webpage? :)

**Comment by asadrahimi** [comment]
If you want to add the mentioned information to your web page, you can use tags;

<p>one line summary</p>
<br><br>
<h2>Press posts</h2>
<date></date>
<br><br>
<h2>Awards</h2>


I hope this is somehow helpful and addressing your concerns.

**Comment by nishadsinghi** [comment]
Should I add this information in the .bib file or in the publications.md file? if it's in the .bib file, then where exactly? I can't see how to add it to the publications.md file, since it imports information from the .bib file and doesn't contain information about the publications themselves.

**Comment by george-gca** [comment, marked as answer]
I added a `TLDR` for my publications, as you can see in my [page](https://george-gca.github.io/publications/). For this, search for `tldr` in my [_layouts/bib.html](https://github.com/george-gca/george-gca.github.io/blob/gca/_layouts/bib.html), [_bibliography/papers.bib](https://github.com/george-gca/george-gca.github.io/blob/gca/_bibliography/papers.bib), and [_sass/_base.scss](https://github.com/george-gca/george-gca.github.io/blob/gca/_sass/_base.scss).

**Reply by Ir1d** [reply]
For future reference, there are a couple of other things to edit. 
<img width="333" alt="image" src="https://github.com/alshedivat/al-folio/assets/10709657/335b70ca-3a78-4a82-9b37-db92d28c46b3">

**Reply by haukesand** [reply]
<img width="821" alt="Bildschirmfoto 2024-10-15 um 8 16 31 PM" src="https://github.com/user-attachments/assets/baeeb556-d771-4228-961a-8ff21b14c065">


I tried another approach. Rendering the TLDR right on the front page without another button click. 

--- 
1. First, add the `tldr` field to your BibTeX entry in the `papers.bib` file:


```94:96:_bibliography/papers.bib
    selected     = {true},
    abbr         = {CHI Workshop},
    tldr         = {Examines the dual role of Generative AI tools in speeding up design iterations but also potentially causing shallow learning in HCI education.}
```


2. Now, we need to modify the `bib.liquid` layout file to include the new TLDR field. We'll add it after the buttons section. Here's the code to add:

```liquid
{% if entry.tldr %}
  <div class="tldr">
    <strong>TLDR:</strong> {{ entry.tldr }}
  </div>
{% endif %}
```

3. To style the TLDR section, we need to add some CSS. Add the following to your `_sass/_base.scss` file:

```scss
.tldr {
  margin-top: 0.5rem;
  margin-bottom: 0.5rem;
  font-size: 0.9rem;
  color: var(--global-text-color-light);
}
```

**Reply by george-gca** [reply]
@haukesand I like your approach. Would you mind sending a PR for this? Also, how does it looks like when some papers have a TLDR and some not? Add an example in just one of the papers of the template.

**Comment by nishadsinghi** [comment]
Hi @george-gca and @lr1d, thanks for your inputs :) What I was asking for is a bit different. I would like to add some text right below the publication, without having to click on another button (for example, some special achievement). Here is an example: https://imgur.com/a/tAblgXI

**Reply by haukesand** [reply]
Yes, I am looking to do the same such as adding an emoji badge  for best paper award, and honorable mentions 🏆

**Reply by george-gca** [reply]
Currently we support the field `additional_info` as can be seen in [_bibliography/papers.bib](https://github.com/alshedivat/al-folio/blob/de4e89d11b44ca2b1660aeeafde5e88b6415542f/_bibliography/papers.bib#L58C3-L58C18) and used in our [publication list](https://alshedivat.github.io/al-folio/publications/).

**Reply by haukesand** [reply]
Thanks. I chose to use the abbreviation field to add a small emoji   abbr  ={🏆 CHI}
<img width="159" alt="Screenshot 2024-08-26 at 11 42 50 PM" src="https://github.com/user-attachments/assets/c9bf6ee4-ff4a-44da-a1b8-4d0f56350ebe">

**Comment by george-gca** [comment]
You should add this information somewhere to the publications as a new field in [_bibliography/papers.bib](https://github.com/alshedivat/al-folio/blob/master/_bibliography/papers.bib), then display this information in [_layouts/bib.liquid](https://github.com/alshedivat/al-folio/blob/master/_layouts/bib.liquid), which is the file responsible for creating the individual publications views.

Check for example how the `title`, `year`, or `authors` information is handled in both files.
